data/net/minecraft/network/protocol/game/ClientboundSetEntityLinkPacket.mapping

```diff
@@ -1,9 +1,14 @@
 CLASS net/minecraft/network/protocol/game/ClientboundSetEntityLinkPacket
 	FIELD destId I
 		COMMENT The entity that is holding the leash, or -1 to clear the holder.
+	METHOD <init> (Lnet/minecraft/network/FriendlyByteBuf;)V
+		ARG 1 byteBuf
+	METHOD <init> (Lnet/minecraft/world/entity/Entity;Lnet/minecraft/world/entity/Entity;)V
+		ARG 1 source
+		ARG 2 destinationOrNull
```

Comment:
Might want to leave this as `destination` and relegate nullability to a javadoc comment.